mozilla_schema_generator/probes.py

```diff
@@ -51,39 +51,8 @@ class MainProbe(Probe):
 
     first_added_key = "first_added"
 
-    # TODO: Do we use log_sum_*, sum_squares_*?
     histogram_schema = {
-        "type": "object",
-        "properties": {
-            "bucket_count": {
-                "minimum": 0,
-                "type": "integer"
-            },
-            "histogram_type": {
-                "minimum": 0,
-                "type": "integer"
-            },
-            "range": {
-                "items": {
-                    "type": "integer"
-                },
-                "type": "array"
-            },
-            "sum": {
-                "minimum": 0,
-                "type": "integer"
-            },
-            "values": {
-                "additionalProperties": False,
-                "patternProperties": {
-                    "^[0-9]+$": {
-                        "minimum": 0,
-                        "type": "integer"
-                    }
-                },
-                "type": "object"
-            }
-        }
+        "type": "string",
```

Comment:
This appears to be the only change needed in the actual code here?

In order to verify this made the BQ schema I wanted, I had to pipe the json schema output through jq:

```
mozilla-schema-generator generate-main-ping | jq -r '.full[0]' > histo_string.json
jsonschema-transpiler -t bigquery histo_string.json > histo_string.bq
```

I assume that the bin/schema_generator.sh machinery handles parsing out the top level in the produced `.json` file?

Reply:
This should be the only modification that's needed, I'm not sure about the test though.

It sounds like you have an out of date version of jsonschema-transpiler, as of v1.0, the BigQuery schemas [return the root as a list](https://github.com/mozilla/jsonschema-transpiler/releases/tag/1.0.0). You can upgrade using `cargo install jsonschema-transpiler --force`. It also reads directly from stdin, so you can pipe everything in a single line.

Reply:
I'm on jsonschema-transpiler 1.0 and it's doing what I expect; it's m-s-g that's putting out a json file that starts with `{"full": [ ...` that I'm not expecting.

Reply:
Interesting, I don't think we're using the generated main ping schema yet. It looks like the schema_generator script needs to be modified accordingly to get it in the daily updates. 

Maybe around or before here?

https://github.com/mozilla/mozilla-schema-generator/blob/a24083c9b77a77d949ba859c9eb7db2e6b8ea7cc/bin/schema_generator.sh#L108-L110